src/panels/lovelace/common/compute-tooltip.ts

```diff
@@ -6,11 +6,16 @@ import { ActionConfig } from "../../../data/lovelace";
 interface Config extends LovelaceElementConfig {
   entity?: string;
   title?: string;
+  noTitle?: boolean;
```

Comment:
camelcase is inconsistent with how we define all other configuration options. I think `show_tooltip` is a better descriptor.

Reply:
Ooops.
show is the default behavior so I don't want to change that. I changed it to hide_title.

Reply:
`show` is the common phrase currently used for things like this. e.g. https://www.home-assistant.io/lovelace/picture-entity/#show_name

Reply:
Done

Reply:
We should just allow setting `title` to `null` to indicate no title should be shown.

Reply:
Didn't think that was such good practice. Anyway it's easier so done.